package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "format": "prettier --write \"**/*.+(ts|tsx)\"",
```

Comment:
I added a `precommit` hook using lint-staged to run prettier on the staged files. Try `npm install` again and you won't need to run `format` manually

Reply:
Will do. Should it also run prettier on save?